### PR TITLE
Rename procfile target

### PR DESCRIPTION
**Pull request by robacarp**
This is a compliment to https://github.com/luckyframework/heroku-buildpack-crystal/pull/11

See https://github.com/luckyframework/heroku-buildpack-crystal/pull/11#issuecomment-770074711

I also added a `target` stanza to the shard.yml. It's an out of context update and I'm happy to pull the commit if needed.

### CODE REVIEW

File: src/web_app_skeleton/Procfile.ecr [hunk 1]
```diff
@@ -0,0 +1,2 @@
+web: bin/<%= project_name %>
```
Comment (jwoertink) [inline src/web_app_skeleton/Procfile.ecr:1]: Let's add a spec line https://github.com/luckyframework/lucky_cli/blob/master/spec/integration/init_web_spec.cr#L19 that looks for `Procfile` to contain `bin/test-project`

File: shard.yml [hunk 2]
```diff
@@ -4,6 +4,10 @@ version: 0.25.0
 authors:
   - Paul Smith <paulcsmith0218@gmail.com>
 
+targets:
+  lucky:
+    main: src/lucky.cr
```
Comment (jwoertink) [inline shard.yml:9]: 👍

**Review by jwoertink — CHANGES_REQUESTED** [review]
I'm not sure if I've ever seen the deploy to heroku specs run or not.. Will be interesting to see if they run and pass. I think just a little spec to ensure that's good, and we can merge this in.

File: src/web_app_skeleton/Procfile.ecr (hunk 1 above)
Comment (robacarp) [reply src/web_app_skeleton/Procfile.ecr:1]: Good idea!

**Review by jwoertink — APPROVED** [review]
Let's roll with it!

**Pull request merged**